climada/engine/impact.py

```diff
@@ -510,6 +514,79 @@ def write_sparse_csr(self, file_name):
         np.savez(file_name, data=self.imp_mat.data, indices=self.imp_mat.indices,
                  indptr=self.imp_mat.indptr, shape=self.imp_mat.shape)
 
+    def write_netcdf(self, path=None, attrs=None, with_imp_mat=True, **kwargs):
+        """Write this instance into a (rasterized) netcdf via a DataArray.
+
+        Parameters:
+            path (str, optional): Where to write the resulting netCDF to. If
+                None, the method stands in as a to_dataset
+            attrs (str, optional): Set as an attribute; Dataset.to_netcdf does
+                not handle dimensionless variables properly, so GDAL cannot
+                handle contained projection information. Still worthwhile for
+                human reference though.
+            with_imp_mat (bool): If true, read impact values from imp_mat
+                instead of eai_exp
```

Comment:
This is incomprehensible, especially without implementation knowledge but even when I look at the code it's not clear to me.

Reply:
Yes, that is problematic. I'll translate it. To even have an imp_mat, one must set `save_mat` to true in Impact.calc, so I assumed that people would have knowledge. I suppose that argument will have to remain for now, but it could also be renamed for comprehension's sake.